### PR TITLE
Added Player::SessionProperties

### PR DESCRIPTION
**Pull request by TrickyBestia**

### CODE REVIEW

**Comment by Thundermaker300** [comment]
Would we want methods for this (eg. `Ply.AddCustomData(string, object)`, `Ply.GetCustomData(string, object)`, etc)? I've seen some people mention it

**Comment by TrickyBestia** [comment]
@Thundermaker300 `Dictionary` provides us the same methods

**Comment by Thundermaker300** [comment]
Yeah I know, wasn't sure if we wanted to give direct access to the dictionary or provide methods that do it

File: Exiled.API/Features/Player.cs [hunk 1]
```diff
@@ -242,6 +242,11 @@ public string CustomPlayerInfo
             set => ReferenceHub.nicknameSync.Network_customPlayerInfoString = value;
         }
 
+        /// <summary>
+        /// Gets the dictionary of player's session properties. It is not being saved on player disconnect.
+        /// </summary>
+        public Dictionary<string, object> SessionProperties { get; } = new Dictionary<string, object>();
```
Comment (iRebbok) [inline Exiled.API/Features/Player.cs:248]:
```suggestion
        /// <summary>
        /// Gets the dictionary of player's session variables. It is not being saved on player disconnect.
        /// </summary>
        public Dictionary<string, object> SessionVariables { get; } = new Dictionary<string, object>();
```

**Comment by iRebbok** [comment]
Can't say that this is very very useful, but the plugin devs may use this to tell a plugin that the player is (for example) SCP-035 or something.

**Pull request merged**